### PR TITLE
Feat endpoint mock

### PR DESCRIPTION
**Pull request by marcambier**
**Issue**

https://github.com/gravitee-io/issues/issues/8229

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/feat-endpoint-mock/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

File: gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-handler/src/main/resources/META-INF/spring.factories [hunk 1]
```diff
@@ -0,0 +1,2 @@
+io.gravitee.plugin.core.api.PluginHandler=\
+io.gravitee.plugin.endpoint.endpoint.internal.EndpointPluginHandler
```
Comment (marcambier) [inline gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-handler/src/main/resources/META-INF/spring.factories]: wrong package and class name

File: gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-mock/src/test/java/io/gravitee/plugin/endpoint/mock/DummyMessage.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import io.gravitee.gateway.api.buffer.Buffer;
+import io.gravitee.gateway.api.http.HttpHeaders;
+import io.gravitee.gateway.jupiter.api.message.Message;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author GraviteeSource Team
+ */
+public class DummyMessage implements Message {
```
Comment (marcambier) [inline gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-mock/src/test/java/io/gravitee/plugin/endpoint/mock/DummyMessage.java]: useless class, to remove

File: gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-sse/src/main/resources/plugin.properties [hunk 3]
```diff
@@ -2,5 +2,5 @@ id=sse
 name=Entrypoint SSE
 version=${project.version}
 description=${project.description}
-class=io.gravitee.plugin.entrypoint.sse.SseEntrypointConnectorFactory
-type=entrypoint
+class=main.java.io.gravitee.plugin.endpoint.mock.SseEntrypointConnectorFactory
```
Comment (marcambier) [inline gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-sse/src/main/resources/plugin.properties]: wrong package and class

File: gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-mock/src/main/resources/schemas/schema-form.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "type": "object",
+  "id": "urn:jsonschema:io:gravitee:plugin:endpoint:mock:configuration:MockEndpointConnectorConfiguration",
+  "properties": {},
```
Comment (marcambier) [inline gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-mock/src/main/resources/schemas/schema-form.json:4]: missing properties list

**Review by jhaeyaert — APPROVED** [review]

**Pull request merged**